ci-operator/step-registry/sandboxed-containers-operator/README.md

```diff
@@ -0,0 +1,57 @@
+This directory contain the steps, chains and workflows implemented specifically for the Openshift Sandboxed Containers (OSC) jobs.
+
+## Steps
+
+Here is the list of steps and their explanation.
+
+Please refer to the `*-ref.yaml` file in their source code for the full list of parameters accepted by each step.
+
+### sandboxed-containers-operator-get-kata-rpm
+
+The [sandboxed-containers-operator-get-kata-rpm](./get-kata-rpm/) step downloads the kata-containers rpm from Brew and copy it over the cluster worker nodes.
+
+### sandboxed-containers-operator-peerpods-param-cm
+
+The [sandboxed-containers-operator-peerpods-param-cm](./peerpods/param-cm/) step creates the peerpods-param-cm configmap. Currently only Azure is supported and it will do the needed networking setup for OSC to work properly on this cloud provider.
```

Comment:
You use very long lines, it'd be nice to break them (if not at 80 chars at least around 120/140 chars), which should decrease the diffs in case of updates.